airflow/www/static/js/dag/details/dagRun/ClearRun.tsx

```diff
@@ -59,31 +60,57 @@ const ClearRun = ({ runId, ...otherProps }: Props) => {
     onQueue({ confirmed: true });
   };
 
-  useKeysPress(keyboardShortcutIdentifier.dagRunClear, clearExistingTasks);
+  const [showConfirmationModal, setShowConfirmationModal] = useState(false);
+  const [doNotShowAgain, setDoNotShowAgain] = useState(false);
```

Comment:
We should store the do not show again in localstorage for it to be useful. Otherwise it will be reset on a page refresh. If not, we should just remove it entirely.

Reply:
I would still like to update this before approving the PR.

Reply:
Yes, I'm working on this.

Reply:
Stored the do not show again in local storage. Took some time as was having trouble with testing the functionality.